src/docs/grid-aware-websites/tutorials/grid-aware-tutorial-cloudflare-workers.md

```diff
@@ -39,7 +39,7 @@ You should also be aware of the limits and pricing of Cloudflare Workers, availa
 
 ## Creating a new Cloudflare Workers project
 
-To begin using Grid-aware Websites on an existing website through Cloudflare Workers, we will first create a new Cloudflare Workers project on our development machine. You can do that by fellowing the steps below, or by visiting the [Cloudflare Workers documentation](https://developers.cloudflare.com/workers/get-started/guide/).
+To begin using Grid-aware Websites on an existing website through Cloudflare Workers, we will first create a new Cloudflare Workers project on our development machine. You can do that by following the steps below, or by visiting the [Cloudflare Workers documentation](https://developers.cloudflare.com/workers/get-started/guide/).
 
 In a new terminal window, run the command below. We've called our project `grid-aware-worker`, but you can give it whatever name you like.
```

Comment:
A few lines below outside the diff:

> Running this command will present you with a series of prompts to setup your Cloudflare Worker project.

I think it would be nice to roughly say how to answer the prompts? Just to reduce the uncertainty for people going through this.